tasks/ast_codegen/src/schema/defs.rs

```diff
@@ -120,13 +128,18 @@ pub struct InheritDef {
 pub struct FieldDef {
     /// `None` if unnamed
     pub name: Option<String>,
+    #[serde(skip)]
     pub vis: Visibility,
+    #[serde(rename = "type")]
     pub typ: TypeRef,
+    #[serde(skip)]
```

Comment:
We have to serialize markers to pass the information about serde attributes(it would be part of the markers).
I've used `#[serde(skip)]` at the top level for `OuterMarkers because of the `syn` types used there. We have to turn them into primitives.

Reply:
I'm just trying to reduce noise in the schema at the moment so I can see what (if any) other changes would be needed for it to be consumable by AST transfer. Let's remove `#[serde(skip)]` again once the info from serde attrs is in the schema.

I'm also unclear what the criteria for information living in `markers` is (as opposed to being a property on the main object).

Reply:
Fields already have their own markers.